zkevm-circuits/src/evm_circuit/op_execution/math.rs

```diff
@@ -0,0 +1,215 @@
+use super::super::Cell;
```

Comment:
math.rs file name seem too generic

Reply:
Yes agreed, currently changed it to `math_gadgets` which is hopefully at least a bit better. :)